src/tasks/python/PythonExtensionHelper.ts

```diff
@@ -19,6 +17,23 @@ export namespace PythonExtensionHelper {
     }
 
     export async function getLauncherFolderPath(): Promise<string> {
+        const pyExt = await getPythonExtension();
+
+        /* eslint-disable @typescript-eslint/tslint/config */
+        if (pyExt?.exports?.debug) {
+            const debuggerPath = await pyExt.exports.debug.getDebuggerPackagePath();
+
+            if (debuggerPath) {
+                return debuggerPath;
+            }
+        }
+        /* eslint-enable @typescript-eslint/tslint/config */
+
+        throw new Error(localize('vscode-docker.tasks.pythonExt.noDebugger', 'Unable to find the debugger in the Python extension.'));
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    export async function getPythonExtension(): Promise<vscode.Extension<any>> | undefined {
```

Comment:
We should make an interface file containing their extension API. It won't be any more susceptible to out-of-sync issues than we already are, but would let us get rid of suppressions and make it clearer what the Python extension is giving us.

Reply:
It's fine to include only the parts of the API we are using. We should also have a link in it to their source file.